index.js

```diff
@@ -26,10 +26,16 @@ module.exports = (function () {
    * @return {Promise} returns String passage
    */
   bible.get = function (psg, ver) {
-    if (typeof psg !== 'string') { throw new TypeError('Expected a string'); }
+    if (typeof psg !== 'string') {
+      throw new TypeError('Passage should be a string');
+    }
+    if (typeof ver !== 'undefined' && typeof ver !== 'string') {
+      throw new TypeError('Version should be a string');
+    }
 
     var self = this;
     var v = ver || 'asv';
```

Comment:
That's avoided now that you added the check above, if ver is undefined, throw an error :)

With this approach, the pipes might be useless actually. Because `ver` will always be defined, but we actually want to check that it's a version that is supported. We should probably add a check to see if version is supported instead of double pipe.